test/browser-providers.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+/*
+ * Browser Configuration for the different jobs in the CI.
+ * Target can be either: BS (Browserstack) | SL (Saucelabs) | null (To not run at all)
+ */
+const browserConfig = {
+  'Chrome':       { unitTest: {target: 'SL', required: true  }},
+  'Firefox':      { unitTest: {target: 'SL', required: true  }},
+  'ChromeBeta':   { unitTest: {target: 'SL', required: false }},
+  'FirefoxBeta':  { unitTest: {target: null, required: false }},
+  'ChromeDev':    { unitTest: {target: null, required: true  }},
+  'FirefoxDev':   { unitTest: {target: null, required: true  }},
+  'IE9':          { unitTest: {target: null, required: false }},
+  'IE10':         { unitTest: {target: null, required: true  }},
+  'IE11':         { unitTest: {target: 'SL', required: true  }},
+  'Edge':         { unitTest: {target: 'SL', required: true  }},
+  'Android4.1':   { unitTest: {target: null, required: false }},
+  'Android4.2':   { unitTest: {target: null, required: false }},
+  'Android4.3':   { unitTest: {target: null, required: false }},
+  'Android4.4':   { unitTest: {target: 'SL', required: false }},
+  'Android5':     { unitTest: {target: 'SL', required: false }},
+  'Safari7':      { unitTest: {target: null, required: false }},
+  'Safari8':      { unitTest: {target: 'BS', required: false }},
+  'Safari9':      { unitTest: {target: 'BS', required: false }},
```

Comment:
Idle thought: looks like Browserstack still doesn't support Safari 10, but SauceLabs does. I wonder if we should try adding that (in a follow-up PR)